cime_config/testdefs/testmods_dirs/clm/collapse_pfts_78_to_16/user_nl_clm

```diff
@@ -0,0 +1,3 @@
+for_testing_allow_non_annual_changes = .true.
```

Comment:
I don't think this 'allow_non_annual_changes' is needed for this test, and should be removed.

Reply:
Removing.